code/script.js

```diff
@@ -1,325 +1,380 @@
-// All the DOM selectors stored as short variables
-const board = document.getElementById('board')
-const questions = document.getElementById('questions')
-const restartButton = document.getElementById('restart')
+//DOM selectors declared here
+const board = document.getElementById("board");
+const dropdown = document.getElementById("dropdown");
+const playAgainButton = document.getElementById("playAgain");
+const restartButton = document.getElementById("restartBtn");
+const filterButton = document.getElementById("filter");
+const questionCounter = document.getElementById("questions-left");
 
```

Comment:
Starting off with the naming chosen for the DOM selectors; they're clear and easy to follow. Them matching the Id-names also further showcases the clarity and preciseness.